### PR TITLE
Create SIG Recommenders charter

### PR DESCRIPTION
**Pull request by theadactyl**
PTAL @smilingday

### CODE REVIEW

**Comment by smilingday** [comment]
@theadactyl This looks good, thanks for helping!

**Review by ewilderj — APPROVED** [review]
Approving, with a couple of nits:
* please consider reformatting to fit within the usual line-width 
* include details of how people can contact SIG leads. Email address or GitHub handle should suffice.

File: sigs/recommenders/CHARTER.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+* Community leads
+   * Bo Liu, Pinterest
+   * Haidong Rong, Tencent
+   * Yong Li, Alibaba
+* Co-leads and sponsors from Google
+   * Yuefeng Zhou, TensorFlow
+   * Zhenyu Tan, TensorFlow
+   * Wei Wei, TensorFlow
+   * Derek Cheng, Google Brain
```
Comment (smilingday) [inline sigs/recommenders/CHARTER.md:53]: * Community leads
   * Bo Liu, Pinterest, bol@pinterest.com
   * Haidong Rong, Tencent, hudsonrong@tencent.com
   * Yong Li, Alibaba, jiufeng.ly@alibaba-inc.com
* Co-leads and sponsors from Google
   * Yuefeng Zhou, TensorFlow, yuefengz@google.com
   * Zhenyu Tan, TensorFlow, tanzheny@google.com
   * Wei Wei, TensorFlow, weiwe@google.com
   * Derek Cheng, Google Brain, zcheng@google.com

Comment (smilingday) [reply sigs/recommenders/CHARTER.md:53]: Ass Edd's comments, I added emails for each SIG leads. Also we ask people to send emails to sig-recommenders@tensorflow.org.

**Comment by smilingday** [comment]
@theadactyl ready to summit this? Thanks!

**Pull request merged**